pkg/cmd/status/status.go

```diff
@@ -677,32 +677,28 @@ func statusRun(opts *StatusOptions) error {
 	section := func(header string, items []StatusItem, width, rowLimit int) (string, error) {
 		tableOut := &bytes.Buffer{}
 		fmt.Fprintln(tableOut, cs.Bold(header))
-		//nolint:staticcheck // SA1019: utils.NewTablePrinterWithOptions is deprecated: use internal/tableprinter
-		tp := utils.NewTablePrinterWithOptions(opts.IO, utils.TablePrinterOptions{
-			IsTTY:    opts.IO.IsStdoutTTY(),
-			MaxWidth: width,
-			Out:      tableOut,
-		})
 		if len(items) == 0 {
-			tp.AddField("Nothing here ^_^", nil, nil)
-			tp.EndRow()
+			fmt.Fprintln(tableOut, "Nothing here ^_^")
 		} else {
+			//nolint:staticcheck // SA1019: Headers distract from numerous nested tables.
+			tp := tableprinter.NewWithWriter(tableOut, opts.IO.IsStdoutTTY(), width, cs, tableprinter.NoHeaders)
 			for i, si := range items {
 				if i == rowLimit {
 					break
 				}
-				tp.AddField(si.Identifier, nil, idStyle)
+				// TODO: Duplicate behavior of old util.TablePrinter: never truncate first column or URLs.
```

Comment:
Does this still need to be addressed?

Reply:
Probably, to keep it consistent. I'll find everywhere that was using the old `util.TablePrinter` and make sure, where needed, a truncation function is passed that returns URLs as-is.

Reply:
I added a `text.TruncateNonURL` truncation function to any field that would support and may contain a URL prefix using the old condition.